api/base/throttling.py

```diff
@@ -101,6 +101,11 @@ class SendEmailThrottle(BaseThrottle, UserRateThrottle):
 
     scope = 'send-email'
 
+    def allow_request(self, request, view):
+        if request.method == 'GET':
```

Comment:
I'm wondering if we should broaden this so 'OPTIONS' request method is included - 

```python
from rest_framework import permissions

def allow_request(self, request, view):
      if request.method in permissions.SAFE_METHODS:
            return True
```

Reply:
Oh yes, totally agree! And Ember does an `OPTIONS` request first so this could definitely be an issue.

Reply:
Updated!